src/codegen/compile.ml

```diff
@@ -1534,7 +1628,7 @@ module BitTagged = struct
      tell it apart from a pointer by looking at the last bits: if set, it is a
      pointer.
 
-     Small here means -2^30 ≤ x < 2^30, and untagging needs to happen with an
+     Small here means -2^sbits ≤ x < 2^sbits, and untagging needs to happen with an
```

Comment:
For unsigned fix-sized numbers, the range of "small" values would be `-2^ubits <= x < 2^ubits`?
I believe, the comment below maybe needs be a adjusted a little bit, e.g. "Note that {Nat,Int}{8,16} do not need to be explicitly bit-tagged" would no longer be valid.

Reply:
For unsinged, fixed size, its `0 <= x < 2^ubits', I think

I've adjust the comment (which shows how the complexity has grown a fair bit, sigh)